### PR TITLE
RavenDB-13789 show information about cloud backup-storage-type after …

### PR DESCRIPTION
**Pull request by Danielle9897**
…entering credentials

### CODE REVIEW

**Comment by Danielle9897** [comment]
This PR needs the following **Cloud** PR:
`https://github.com/hibernating-rhinos/cloud.ravendb.net/pull/310`

File: src/Raven.Studio/wwwroot/App/views/resources/createDatabase.html [hunk 1]
```diff
@@ -300,12 +300,63 @@ <h3>Backup Source Configuration</h3>
...
-            <div class="col-md-8">
-                 <input class="form-control" data-bind="textInput: backupLink" />
+            <div data-bind="if: $root.databaseModel.spinners.backupCredentialsLoading()">
+                <div class="form-group row">
+                    <div class="col-md-offset-4 col-md-8">
+                        <label class="control-label global-spinner spinner-sm"></label>
+                    </div>
+                </div>
+            </div>
+            <div data-bind="if: !$root.databaseModel.spinners.backupCredentialsLoading() && cloudCredentials() && backupLink.isValid()">
```
Comment (grisha-kotler) [inline src/Raven.Studio/wwwroot/App/views/resources/createDatabase.html]: You can use: "with: cloudCredentials" in the parent div

File: src/Raven.Studio/wwwroot/App/views/resources/createDatabase.html [hunk 2]
```diff
@@ -300,12 +300,63 @@ <h3>Backup Source Configuration</h3>
...
+                <div class="form-group row">
+                    <div class="col-md-4">
+                        <label class="control-label">Backup Storage Type</label>
+                    </div>
+                    <div class="col-md-8">
+                        <label class="control-label text-info" data-bind="text: cloudCredentials().BackupStorageType"></label>
```
Comment (grisha-kotler) [inline src/Raven.Studio/wwwroot/App/views/resources/createDatabase.html]: Same here

File: src/Raven.Studio/wwwroot/App/views/resources/createDatabase.html [hunk 3]
```diff
@@ -300,12 +300,63 @@ <h3>Backup Source Configuration</h3>
...
+                    <div class="col-md-8">
+                        <label class="control-label text-info text-left" data-bind="text: cloudCredentials().RemoteFolderName"></label>
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <div class="col-md-4">
+                        <label class="control-label">Link Expiration (Local)</label>
+                    </div>
+                    <div class="col-md-8">
+                        <label class="control-label text-info" data-bind="text: cloudCredentials().Expires"></label>
```
Comment (grisha-kotler) [inline src/Raven.Studio/wwwroot/App/views/resources/createDatabase.html]: Can this be like in the about page?
"in 17 minutes" (tooltip: the local date)

**Review by grisha-kotler — CHANGES_REQUESTED** [review]

File: src/Raven.Studio/wwwroot/App/views/resources/createDatabase.html [hunk 4]
```diff
@@ -300,12 +300,63 @@ <h3>Backup Source Configuration</h3>
...
+                    <div class="col-md-4">
+                        <label class="control-label">Backup Storage Type</label>
+                    </div>
+                    <div class="col-md-8">
+                        <label class="control-label text-info" data-bind="text: cloudCredentials().BackupStorageType"></label>
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <div class="col-md-4">
+                        <label class="control-label">Aws Region Name</label>
```
Comment (ppekrol) [inline src/Raven.Studio/wwwroot/App/views/resources/createDatabase.html]: Remove 'Aws' user will know that from 'Backup Storage Type', right?

File: src/Raven.Studio/wwwroot/App/views/resources/createDatabase.html (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/resources/createDatabase.html]: done

File: src/Raven.Studio/wwwroot/App/views/resources/createDatabase.html (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/resources/createDatabase.html]: done

File: src/Raven.Studio/wwwroot/App/views/resources/createDatabase.html (hunk 4 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/resources/createDatabase.html]: Removed 'Aws'

**Review by ppekrol — APPROVED** [review]

**Pull request merged**